Packaging/ps4/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+SCRIPTDIR="${BASH_SOURCE[0]}"
+SCRIPTDIR="$(dirname "${SCRIPTDIR}")"
+
+cmake -S "${SCRIPTDIR}/../../" \
+      -B build \
```

Comment:
It may be better to build in `build-ps4`, similar to how we do it for other platforms